wlutil/init.py

```diff
@@ -0,0 +1,36 @@
+import logging
+from .wlutil import *
+from .br import br
+
+# Initialize wlutil for the first time
+def oneTimeInit():
+    log = logging.getLogger()
+
+    # Build initramfs fs structure (git can't save these because reasons)
+    initramfs_dirs = ["bin", 'dev', 'etc', 'proc', 'root', 'sbin', 'sys', 'usr/bin', 'usr/sbin', 'mnt/root']
+    for d in initramfs_dirs:
+        if not (initramfs_root / d).exists():
+            (initramfs_root / d).mkdir(parents=True)
+
+    # Make busybox (needed for the initramfs)
+    log.info("Building busybox (used in initramfs)")
```

Comment:
I could use the busybox from buildroot, but then we're beholden to them. I feel safer having a separate submodule just for this that we control. When I started messing with the buildroot config my images got broken (never figured out why).